go.mod

```diff
@@ -7,15 +7,15 @@ require (
 	collectd.org v0.3.0
 	github.com/BurntSushi/toml v0.3.1
 	github.com/apache/arrow/go/arrow v0.0.0-20191024131854-af6fa24be0db
+	github.com/benbjohnson/tmpl v1.1.0 // indirect
```

Comment:
Can we make sure that this import is in `tools.go`? Guessing from the `// indirect` that it's not there.

Reply:
Good catch - thanks! In fact 1.8 doesn't have a tools.go at all, added one as a copy of 1.9